cmake-modules/TSPCompile.cmake

```diff
@@ -0,0 +1,39 @@
+find_package(Git)
+
+macro(DownloadTSPFiles TSP_REPO TSP_BRANCH TSP_REPO_PATH TSP_DESTINATION)
+    message ("Downloading TSP files from ${TSP_REPO} branch ${TSP_BRANCH} path ${TSP_REPO_PATH} to ${TSP_DESTINATION}")
+
+    if(Git_FOUND)
+        message("Git found: ${GIT_EXECUTABLE}")
+    else()
+        message(FATAL_ERROR "Git not found")
+    endif()
```

Comment:
Worth considering the developer experience. When an Azure service team adds the values to the library and gets them wrong. Will the tools fail in a way that makes sense and is easy to diagnose?